src/ir/module-utils.h

```diff
@@ -442,6 +443,22 @@ template<typename T> struct CallGraphPropertyAnalysis {
   }
 };
 
+enum class TypeInclusion { AllTypes, UsedIRTypes, BinaryTypes };
+
+enum class ClassifyAction { NoClassify, Classify };
+
+enum class TypeClassification { Unknown, Public, Private };
```

Comment:
Please add comments for these enum values (e.g. UsedIRTypes and Classify are not trivial).